### DISCUSSION TITLE
Neo Strafe / Tap Strafe / Super Glide Support

### DISCUSSION THREAD

**Discussion by RoastyToast** [cafali/SnapKey#83] [New Features & Ideas]
While looking at the software and using it for a while I was wondering if it would be possible to support the ability to setup macros for Neo Strafe, Tap Strafe and Super Glide based on Keyboard and Mouse key presses.

A lot of people used to use JoyToKey for this in Apex but SnapKey would be able to do this exact same thing and at the same time use the SnapKey functionality for Snap Tapping.

Would love to know if this would be possible :)

**Comment by LightBeamOvHe** [comment]
Macros would definitely be a neat addition, for now there's Free Snap Tap that does it but it's much more « barebones » at least in how it looks, you might want to take a look into that if you want to setup macros right now (I know that you can also setup delays into it to avoid anticheats or whatever, and unless there are reasons to set delays other than avoiding anticheats ... SnapKey will probably never do it which is fair)
